frontend/src/pages/ProjectSettings/FilterExtensionForm/FilterExtensionForm.tsx

```diff
@@ -49,7 +49,8 @@ export const FilterExtensionForm = () => {
 									? {
 											projectSettings: {
 												...currentProjectSettings.projectSettings,
-												filterChromeExtension: isOptIn,
+												filter_chrome_extension:
```

Comment:
This fixes a bug whereby we couldn't change this setting. 
@eightants - Not super pressing but it would be helpful if the `setAllProjectSettings` function had stronger typing to prevent this issue.